src/wutils_app.erl

```diff
@@ -36,3 +37,19 @@ stop(_State) ->
 %%====================================================================
 %% Internal functions
 %%====================================================================
+
+start_pools() ->
+    case application:get_env(pools) of
+        {ok, Pools} ->
+            Errors = lists:filter(fun ({V,_}) ->
+                                          V =:= error
+                                  end, episcina:start_pools(Pools)),
+            case Errors of
+                [] ->
+                    ok;
+                Errs ->
+                    throw(Errs)
```

Comment:
`[] = lists:filter(...)` feels clearer to me. Or maybe `true = lists:all(fun ({V,_}) -> V =/= error end, episcina:start_pools(Pools))`